spec/controllers/companies_controller_spec.rb

```diff
@@ -0,0 +1,168 @@
+require "rails_helper"
+include SessionsHelper
+
+describe CompaniesController, type: :controller do
+  let!(:account_company){FactoryBot.create :account, role: "company"}
```

Comment:
phân biệt cho c let và let!
k nên lạm dụng let! nhé

Reply:
let và let! đều được sử dụng để định nghĩa một biến, trong đó:
**let chỉ chạy khi được gọi đến, và sẽ cached lại giá trị cho những lần sử dụng sau trong cùng một khối it, nhưng ra khỏi khối it thì nó sẽ được khởi tạo lại
**let! sẽ khởi tạo ngay giá trị khi nó được khai báo, giá trị này sẽ được dùng cho toàn bộ test case